node/src/Pos/Wallet/Web/State/Acidic.hs

```diff
@@ -122,8 +121,6 @@ makeAcidic ''WalletStorage
     , 'WS.addOnlyNewTxMetas
     , 'WS.setWalletTxHistory
     , 'WS.addOnlyNewTxMeta
-    , 'WS.removeWallet
-    , 'WS.removeTxMetas
```

Comment:
Hmm, I'm not sure we can remove these. If these transactions were used in any previous release then there can be DB log files out there that use them, and which may still need to be replayed. We can mark them as deprecated, but I would not remove them without a careful historical audit.